src/app/containers/ConsentBanner/Banner/cookie.amp.jsx

```diff
@@ -0,0 +1,246 @@
+    padding-left: ${GEL_MARGIN_ABOVE_400PX};
+    padding-right: ${GEL_MARGIN_ABOVE_400PX};
+  }
+`;
+
+const BannerPage = styled.div`
+  margin-left: auto;
+  margin-right: auto;
+  max-height: 100vh;
+  max-width: ${GEL_GROUP_2_SCREEN_WIDTH_MAX};
+  overflow-y: auto;
+  padding-top: ${GEL_SPACING_DBL};
+
+  @media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) {
+    padding-top: ${GEL_SPACING_QUAD};
+  }
+
+  a {
+    color: ${C_CONSENT_ACTION};
+    text-decoration: underline;
+    text-decoration-color: ${C_PEBBLE};
+  }
+`;
+
+const Title = styled.h2`
+  ${({ script }) => getDoublePica(script)}
+  ${COMMON_HEADING_STYLES}
+`;
+
+const Heading = styled.h3`
+  ${COMMON_HEADING_STYLES}
+`;
+
+const Paragraph = styled.p`
+  ${({ script }) => getBodyCopy(script)}
+  color: ${C_CONSENT_CONTENT};
+  margin-top: ${GEL_SPACING_DBL};
+  margin-bottom: ${GEL_SPACING_DBL};
+`;
+
+const OptionsList = styled.ul`
+  ${({ script }) => getLongPrimer(script)}
+  align-items: stretch;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  list-style: none;
+  margin-top: 0;
+  margin-bottom: 0;
+  padding-right: 0;
+  padding-bottom: ${GEL_SPACING};
+  padding-left: 0;
+
+  @media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) {
+    flex-direction: row;
+    padding-top: ${GEL_SPACING};
+    padding-bottom: ${GEL_SPACING_TRPL};
+  }
+`;
+
+const OptionsItem = styled.li`
+  margin-bottom: ${GEL_SPACING};
+
+  @media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) {
+    width: calc(50% - ${GEL_SPACING});
+  }
+
+  a,
+  button {
+    ${({ service }) => getSansBold(service)}
+    cursor: pointer;
+    display: block;
+  }
+
+  a {
+    padding-top: ${GEL_SPACING};
+    padding-bottom: ${GEL_SPACING};
+    text-align: center;
+  }
+
+  button {
+    ${({ script }) => getLongPrimer(script)}
+    background: ${C_GHOST};
+    border: none;
+    color: ${C_EBON};
+    height: 100%;
+    min-height: ${MIN_TAP_HEIGHT};
+    padding: ${GEL_SPACING};
+    width: 100%;
+  }
+
+  button[on='tap:AMP.setState({ isManagingSettings: true })'] {
+    background: none;
+    border: 0.0625rem solid ${C_CONSENT_ACTION};
+    color: ${C_CONSENT_ACTION};
+  }
+`;
+
+const AmpCookieBanner = ({
+  dir,
+  id,
+  pages,
+  accept,
+  reject,
+  hidden,
+  script,
+  service,
+}) => {
+  const [initial, manage] = pages;
+
+  return (
+    <div id={id} hidden={hidden}>
+      <Helmet>
+        <script
+          async
+          custom-element="amp-bind"
+          src="https://cdn.ampproject.org/v0/amp-bind-0.1.js"
+        />
+      </Helmet>
+      <Wrapper dir={dir} service={service}>
+        <BannerPage data-amp-bind-hidden="isManagingSettings">
+          <Title script={script}>{initial.title}</Title>
+          <Paragraph script={script}>
+            {initial.description.first}{' '}
+            <a href={initial.description.linkUrl}>
+              {initial.description.linkText}
+            </a>{' '}
```

Comment:
Looking at this, I'm not sure why we can't remove these spaces and adjust the translations to add suitable spaces in the `first` and `last` keys; sorry if I'm still missing something?

Reply:
You're not missing anything - I will remove these spaces and update the translations.

Reply:
Did no translations need updating for this change: https://github.com/bbc/simorgh/pull/8718/commits/279b57ba8d052567f8aed9c23689bb17399c1ddd ?

Reply:
From initial tests, strangely not. When I inspected some of the config strings, they already contain the necessary spaces. Hopefully testing will confirm this too.